client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
@ockham Well spotted! I see that the Reader site stream is working correctly given a blog ID (e.g. https://wpcalypso.wordpress.com/read/blogs/33534099) but not a URL. Lookup by URL (e.g. `/read/blogs/developer.wordpress.com`) is a feature that Marin added during his trial but we don't currently use it anywhere in Reader. cc @tyxla